<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatsettingswindow</class>
 <widget class="QDialog" name="chatsettingswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="titleIconLabel">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FCCD4A; color: white; border: 2px solid #FCCD4A; border-radius:19px; padding: 5px 7px;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titleSelectWidget">
        <property name="styleSheet">
         <string notr="true">background-color: white; border: 1px solid lightgray; padding: 5px; border-radius: 10px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="promptWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>15</number>
      </property>
      <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="promptIconLabel">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FCCD4A; color: white; border: 2px solid #FCCD4A; border-radius:19px; padding: 5px 7px;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="promptTitleLabel">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>System prompt:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTextEdit" name="promptSelectWidget">
        <property name="styleSheet">
         <string notr="true">background-color: white; border: 1px solid lightgray; padding: 5px; border-radius: 10px;</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;You are a helpful assistant.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="modelWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="modelIconLabel">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FCCD4A; color: white; border: 2px solid #FCCD4A; border-radius:19px; padding: 5px 7px;</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="modelTitleLabel">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Model:</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QComboBox" name="modelSelectWidget">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>mistralai/Mixtral-8x22B-Instruct-v0.1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mistralai/Mixtral-8x22B-v0.1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mistralai/Mixtral-8x7B-Instruct-v0.1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mistralai/Mistral-7B-Instruct-v0.3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mistralai/Mistral-7B-Instruct-v0.2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mistralai/Mistral-7B-Instruct-v0.1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>meta-llama/Meta-Llama-3-70B-Instruct</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>meta-llama/Meta-Llama-3-8B-Instruct</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>meta-llama/Llama-2-70b-chat-hf</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>meta-llama/Llama-2-13b-chat-hf</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>meta-llama/Llama-2-7b-chat-hf</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>codellama/CodeLlama-70b-Instruct-hf</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>codellama/CodeLlama-34b-Instruct-hf</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>microsoft/WizardLM-2-8x22B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>microsoft/WizardLM-2-7B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>google/gemma-1.1-7b-it</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>google/codegemma-7b-it</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cognitivecomputations/dolphin-2.6-mixtral-8x7b</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>lizpreciatior/lzlv_70b_fp16_hf</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>openchat/openchat-3.6-8b</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>openchat/openchat_3.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>llava-hf/llava-1.5-7b-hf</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>01-ai/Yi-34B-Chat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Austism/chronos-hermes-13b-v2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gryphe/MythoMax-L2-13b-turbo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gryphe/MythoMax-L2-13b</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HuggingFaceH4/zephyr-orpo-141b-A35b-v0.1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Phind/Phind-CodeLlama-34B-v2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>bigcode/starcoder2-15b-instruct-v0.1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>bigcode/starcoder2-15b</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>databricks/dbrx-instruct</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>deepinfra/airoboros-70b</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="ragWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="ragIconLabel">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FCCD4A; color: white; border: 2px solid #FCCD4A; border-radius:19px; padding: 5px 7px;</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ragTitleLabel">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Document:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="ragFileWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white; border: 1px solid lightgray; padding: 5px; border-radius: 10px;</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:15pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ragButton">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttonsWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="submitButton">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
